modules/WordCloudPlugin/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>gephi-plugin-parent</artifactId>
+        <groupId>org.gephi</groupId>
+        <version>0.10.0</version>
+    </parent>
+
+    <groupId>exploreyourdata</groupId>
+    <artifactId>word-cloud-plugin</artifactId>
+    <version>0.10.0</version>
+    <packaging>nbm</packaging>
+    <name>Word Cloud Plugin</name>
+    
+    <properties>
+        <gephi.projectid>org.gephi</gephi.projectid>
+        <gephi.version>0.10.0</gephi.version>
+        <netbeans.version>RELEASE160</netbeans.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>net.clementlevallois.functions</groupId>
```

Comment:
Yes, it works on your local machine but we need to build this plugin in GitHub so it can be released. Without these dependencies in a Maven repository we can't build it. Best would be if your libraries are published to Maven Central (see [guide](https://maven.apache.org/repository/guide-central-repository-upload.html)).

Reply:
Woot? Publishing to Maven central is a big step. Ok, will do.

Reply:
done!